### PR TITLE
feat(sunburst): add mouse events and some labels

### PR DESCRIPTION
**Pull request by juliankrieger**
Because of @mjsarfatti 's absence, I have continued to implement what you recommended @plouc. 

Please review the changes and tell me what you think

-----

- Add slice labels at the first level
- Add onClick/onMouseEnter/onMouseLeave events

### CODE REVIEW

**Comment by mkazlauskas** [comment]
What's holding this from being merged?

**Comment by plouc** [comment]
Thank you @juliankrieger! The build is failing, do you think you could you have a look?

**Comment by juliankrieger** [comment]
Yes @plouc , the build fails because of `45:29  error  'centerRadius' is constant  no-const-assign`. Should be an easy fix.

**Comment by juliankrieger** [comment]
@plouc Build is successful

File: packages/sunburst/src/Sunburst.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
  */
 import React from 'react'
 import PropTypes from 'prop-types'
+import { noop } from '@nivo/core'
```
Comment (wyze) [inline packages/sunburst/src/Sunburst.js:11]: This should be with the other imports from `@nivo/core`.

File: packages/sunburst/src/SunburstLabels.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import { midAngle, positionFromAngle, radiansToDegrees, labelsThemePropType } from '@nivo/core'
+
+const sliceStyle = {
+    pointerEvents: 'none',
+}
+
+export default class SunburstLabels extends Component {
+    static propTypes = {
+        nodes: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+        label: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
```
Comment (wyze) [inline packages/sunburst/src/SunburstLabels.js:20]: Should be updated to just a `func` and be required, based on usage below.

**Comment by juliankrieger** [comment]
I am knee deep in my thesis, maybe after I am done with it.

**Comment by epentin** [comment]
Does anyone have an update on the status of this pr?

**Comment by wyze** [comment]
Thanks everyone for all the work on this and sorry about the long delay on merging!

**Pull request merged**